### PR TITLE
doc:  Add release 2.4.4 to main

### PR DESCRIPTION
**Pull request by umapraseeda**
Add release 2.4.4 to main

### CODE REVIEW

File: doc/nrf/releases_and_maturity/releases/release-notes-2.4.4.rst [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``4bbd91a9083a588002d4397577863e0c54ba7038``.
+It also contains some |NCS| specific additions and commits cherry-picked from the upstream Zephyr repository including the following one:
+
+* Added workarounds in sdk-zephyr for the Anomaly 168 and the Anomaly 165 that are affecting the nRF5340 SoC.
+
+For a complete list of |NCS| specific commits and cherry-picked commits since v2.4.0, run the following command:
+
+.. code-block:: none
+
+   git log --oneline manifest-rev ^v3.3.99-ncs2
```
Comment (umapraseeda) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.4.4.rst:101]: Needs to be updated

File: doc/nrf/releases_and_maturity/releases/release-notes-2.4.4.rst [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+Other libraries
+---------------
+
+* Added :ref:`network_core_monitor` library for monitoring the status of the nRF5340 processor's network core.
+
+Zephyr
+======
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``4bbd91a9083a588002d4397577863e0c54ba7038``.
```
Comment (umapraseeda) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.4.4.rst:92]: SHA Needs to be updated

File: doc/nrf/releases_and_maturity/releases/release-notes-2.4.4.rst [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.4.4`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_244_changelog:
+
+Changelog
+*********
+
```
Comment (umapraseeda) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.4.4.rst:78]: Needs to add the missing sentence.

**Review by gmarull — APPROVED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-2.4.4.rst [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+* Added :ref:`network_core_monitor` library for monitoring the status of the nRF5340 processor's network core.
+
+Zephyr
+======
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``c0aa4d27d3f2767e88c6c8c4634425086156f7f5``.
+It also contains some |NCS| specific additions and commits cherry-picked from the upstream Zephyr repository including the following one:
+
+* Added workarounds in sdk-zephyr for the Anomaly 168 and the Anomaly 165 that are affecting the nRF5340 SoC.
```
Comment (richabp) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.4.4.rst:97]:
```suggestion
* Added workarounds in ``sdk-zephyr`` for the Anomaly 168 and the Anomaly 165 that are affecting the nRF5340 SoC.
```

**Review by richabp — CHANGES_REQUESTED** [review]

**Review by richabp — APPROVED** [review]

**Review by shanthanordic — APPROVED** [review]

**Pull request merged**